FWCore/FWLite/interface/AutoLibraryLoader.h

```diff
@@ -24,6 +24,8 @@ class AutoLibraryLoader {
 private:
   static bool enabled_;
   AutoLibraryLoader();
+
+public:
   AutoLibraryLoader(const AutoLibraryLoader&) = delete;                   // stop default
   const AutoLibraryLoader& operator=(const AutoLibraryLoader&) = delete;  // stop default
```

Comment:
Not a big deal in practice, but I'd prefer these to be moved to the existing `public:` section above. (that would also be more according to our code rules)